build.sbt

```diff
@@ -1,20 +1,23 @@
 
 lazy val commonRootSettings = Seq(
-  version := "1.4.0",
+  version := "1.5.0",
   scalaVersion := "2.11.8",
-  crossScalaVersions := Seq("2.11.8", "2.10.6"),
```

Comment:
Only Play 2.3 is cross compiled, since Play 2.5 does not have a Scala 2.10 or Scala 2.12 version.